includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -64,14 +64,9 @@ public function __construct( WC_Payments_API_Client $payments_api_client, WC_Pay
 		$this->description        = __( 'You will be redirected to Stripe.', 'woocommerce-payments' );
 		$this->payment_methods    = $payment_methods;
 
-		add_action( 'wp_ajax_create_payment_intent', [ $this, 'create_payment_intent_ajax' ] );
-		add_action( 'wp_ajax_nopriv_create_payment_intent', [ $this, 'create_payment_intent_ajax' ] );
-
-		add_action( 'wp_ajax_update_payment_intent', [ $this, 'update_payment_intent_ajax' ] );
-		add_action( 'wp_ajax_nopriv_update_payment_intent', [ $this, 'update_payment_intent_ajax' ] );
-
-		add_action( 'wp_ajax_init_setup_intent', [ $this, 'init_setup_intent_ajax' ] );
-		add_action( 'wp_ajax_nopriv_init_setup_intent', [ $this, 'init_setup_intent_ajax' ] );
+		add_action( 'wc_ajax_wcpay_create_payment_intent', [ $this, 'create_payment_intent_ajax' ] );
+		add_action( 'wc_ajax_wcpay_update_payment_intent', [ $this, 'update_payment_intent_ajax' ] );
+		add_action( 'wc_ajax_wcpay_init_setup_intent', [ $this, 'init_setup_intent_ajax' ] );
```

Comment:
There is no currency used when initialising a setup intent, so I don't think this necessarily needs to be sent in the customer's context, but I've changed it mainly for consistency's sake and perhaps to avoid unforeseen issues down the line. For full disclosure, the function that creates a setup intent to add a payment method is currently still sent in an admin context: that is because this function is also used by the previous payment gateway and I'd rather not tinker where I don't need to.